uat/testing-features/src/main/resources/greengrass/features/shadow-1.feature

```diff
@@ -57,4 +50,61 @@ Feature: Greengrass V2 ShadowManager
+        When I install the component ShadowComponentPing from local store with configuration
+            | key                    | value                                                                                                     |
+            | Operation              | UpdateThingShadow                                                                                         |
+            | ThingName              | testThing                                                                                                 |
+            | ShadowName             | testShadow                                                                                                |
+            | ShadowDocument         | {\"state\":{\"reported\":{\"color\":{\"r\":255,\"g\":255,\"b\":255},\"SomeKey\":\"SomeValue\"}}}          |
+        Then the local Greengrass deployment is SUCCEEDED on the device after 120 seconds
+        When I install the component ShadowComponentPong from local store with configuration
+            | key            | value             |
+            | Operation      | DeleteThingShadow |
+            | ThingName      | testThing         |
+            | ShadowName     | testShadow        |
+        Then the local Greengrass deployment is SUCCEEDED on the device after 120 seconds
+        And I update the component ShadowComponentPing with configuration
+            | key            | value          |
+            | Operation      | GetThingShadow |
+            | ThingName      | testThing      |
+            | ShadowName     | testShadow     |
+        Then the Greengrass deployment is COMPLETED on the device after 2 minutes
+
+    @Shadow1_T4
+    Scenario: Shadow-1-T4: As a developer, I can use the Greengrass local shadow service to LIST Named Shadows from my component.
+        When I install the component ShadowComponentPong from local store with configuration
+            | key                    | value                                                                                                          |
+            | Operation              | SetupListNamedShadowTest                                                                                       |
+            | ThingName              | testThing                                                                                                      |
+            | ShadowDocument         | {\"state\":{\"reported\":{\"color\":{\"r\":255,\"g\":255,\"b\":255},\"SomeKey\":\"SomeValue\"}}}               |
+        Then the local Greengrass deployment is SUCCEEDED on the device after 120 seconds
+        When I install the component ShadowComponentPing from local store with configuration
+            | key       | value                    |
+            | Operation | ListNamedShadowsForThing |
+            | ThingName | testThing                |
+        Then the local Greengrass deployment is SUCCEEDED on the device after 120 seconds
+
+    @Shadow1_T5
```

Comment:
You can remove these tags on each scenario.

Reply:
removed